docs/Analyzer Configuration.md

```diff
@@ -773,3 +773,15 @@ Option Values: Boolean values
 Default Value: `false`
 
 Example: `dotnet_code_quality.CA1051.exclude_structs = true`
+
+### Exclude OrDefault methods
```

Comment:
Can we add some additional description here for what exactly is being excluded? At least we should give an example for clarity.

Reply:
Maybe the title itself can be `Exclude FirstOrDefault and LastOrDefault methods` to match analyzer implementation?